ResearchKit/Common/ORKWebViewStepViewController.h

```diff
@@ -48,7 +48,8 @@ NS_ASSUME_NONNULL_BEGIN
  a web view step in a task, and present a task view controller for that task.
  */
 @interface ORKWebViewStepViewController : ORKStepViewController<WKScriptMessageHandler, WKNavigationDelegate>
-
+@property (nonatomic, strong, readonly) WKWebView *webView;
```

Comment:
Same as above regarding _webView ivar and @property declaration.

Reply:
Updated